sdk/deviceupdate/azure-deviceupdate/azure/deviceupdate/_configuration.py

```diff
@@ -0,0 +1,74 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# --------------------------------------------------------------------------
+
+from typing import TYPE_CHECKING
+
+from azure.core.configuration import Configuration
+from azure.core.pipeline import policies
+
+from ._version import VERSION
+
+if TYPE_CHECKING:
+    # pylint: disable=unused-import,ungrouped-imports
+    from typing import Any
+
+    from azure.core.credentials import TokenCredential
+
+
+class DeviceUpdateClientConfiguration(Configuration):
+    """Configuration for DeviceUpdateClient.
+
+    Note that all parameters used to create this instance are saved as instance
+    attributes.
+
+    :param credential: Credential needed for the client to connect to Azure.
+    :type credential: ~azure.core.credentials.TokenCredential
+    :param account_endpoint: Account endpoint.
+    :type account_endpoint: str
+    :param instance_id: Account instance identifier.
+    :type instance_id: str
+    """
+
+    def __init__(
+        self,
+        credential,  # type: "TokenCredential"
+        account_endpoint,  # type: str
+        instance_id,  # type: str
+        **kwargs  # type: Any
+    ):
+        # type: (...) -> None
+        if credential is None:
+            raise ValueError("Parameter 'credential' must not be None.")
+        if account_endpoint is None:
+            raise ValueError("Parameter 'account_endpoint' must not be None.")
+        if instance_id is None:
+            raise ValueError("Parameter 'instance_id' must not be None.")
+        super(DeviceUpdateClientConfiguration, self).__init__(**kwargs)
+
+        self.credential = credential
+        self.account_endpoint = account_endpoint
+        self.instance_id = instance_id
+        self.credential_scopes = kwargs.pop('credential_scopes', ['6ee392c4-d339-4083-b04d-6b7947c6cf78/.default'])
```

Comment:
I just want to clarify: are your credential scopes really this number path? For reference, most of the credential scopes we see are something like `https://cognitiveservices.azure.com/default`

Reply:
Yeah, this is a valid value; I am following up with the team just for my information why did they pick such a value.